### PR TITLE
Update Readme to link development building to the website wiki. also fix forking and licensing.

### PR DESCRIPTION
**Pull request by ZekeZDev**

### CODE REVIEW

**Review by Scrumplex — APPROVED** [review]

**Review by d-513 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -72,3 +72,9 @@ To modify download infomation or change packaging infomation send a pull request
 ## Forking/Redistributing/Custom builds policy
 
 Do whatever you want, we don't care. Just follow the license. If you have any questions about this feel free to ask in an issue.
+
+All launcher code is available under the GPL-3 license.
```
Comment (txtsd) [inline README.md:76]: Do we want this to say launcher or PolyMC? Launcher is just the de-branded name.

**Comment by WTechNinja** [comment]
While the Readme is being updated, could we update the little "Download Infomation" section to add R's to change the three `infomation`'s to `information`'s? It's a small enough change I don't really want to open a pull request

File: README.md (hunk 1 above)
Comment (txtsd) [reply README.md:76]: @ZekeSmith ^

**Review by timoreo22 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (txtsd) [reply README.md:76]: Bruh, you resolved it without addressing it.

**Pull request merged**